### PR TITLE
Rewrite docs for 0.20.0

### PR DESCRIPTION
**Pull request by bencroker**
This is intended to be a minor rewrite of the docs for the upcoming 0.20.0 release.

### CODE REVIEW

**Comment by bencroker** [comment]
@delaneyj can you please have the docs append an anchor tag to each header element in the page body? 

Input:

```
## Installation
```

Output:

```html
<h2 id="installation">
  Installation <a href="installation">#</a>
</h2>
```

**Review by delaneyj — APPROVED** [review]

File: code/go/site/static/md/guide/getting_started.md [hunk 1]
```diff
@@ -1,46 +1,44 @@
...
-1. Send the current UI from your backend via HTML fragments like HTMX.
-2. Manage client side state that wouldn't make sense to be managed by your backend like AlpineJS.
+With Datastar, you can build any UI that a full-stack framework like React, Vue.js or Svelte can, using a much simpler, hypermedia-driven approach.
 
-I've had [thoughts](/essays/why_another_framework) on both of these in the past. TLDR; While both libraries are great, I wanted to go in a different direction.
-
-Datastar accomplishes both tasks in a unified manner and it's [tiny](https://bundlephobia.com/package/@sudodevnull/datastar).
+<div class="alert">
+    <p>
+        We’re so confident that Datastar can be used as a full-stack framework replacement that we challenge anyone to find a use-case that Datastar _cannot_ handle!
```
Comment (bencroker) [inline code/go/site/static/md/guide/getting_started.md:12]: @delaneyj what do you think, is this too bold?

**Comment by bencroker** [comment]
A language switcher for the SDK example would also be ace 😉

**Comment by bencroker** [comment]
@delaneyj let me know once you’ve committed to `@` and I’ll finalise this PR.

**Pull request merged**